python.d/linux_power_supply.chart.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Description: Linux power_supply netdata python.d module
+# Author: Austin S. Hemmelgarn (Ferroin)
+
+import os
+
+from bases.FrameworkServices.SocketService import SimpleService
```

Comment:
> SocketService import SimpleService

^^

Reply:
Gah, forgot to update that all the way when I copied over the first few lines form another plugin to save time typing...